transform/models/intermediate/detector_health_diag/int_pems__det_diag_high_value_count.sql

```diff
@@ -0,0 +1,87 @@
+with
+source as (
+    select * from {{ source("clearinghouse", "station_raw") }}

```

Comment:
please select from the staging model rather than directly from the source here:
```suggestion
    select * from {{ ref("stg_clearinghouse__station_raw") }}

```